src/libs/runtime-utils.js

```diff
@@ -268,11 +268,21 @@ export const getDiskAppType = disk => {
   return appType
 }
 
+export const projectHasMultipleDisks = disks => {
+  const diskWorkspaces = _.map(currentDisk => currentDisk.labels.saturnWorkspaceName, disks)
+  return _.uniq(diskWorkspaces).length < diskWorkspaces.length
+}
+
+export const projectHasMultipleApps = apps => {
```

Comment:
Same comment about documentation and name as above.

Reply:
Please add unit test coverage for this as well.

Also, the implementation is the same for the 2 methods, so you could consider refactoring them to share the implementation, as long as you can make the contract of what is passed in clear.

(Note though that I'm somewhat in favor of making the methods do the appType filtering, which if you go down that route could make sharing the code more complicated.)